### PR TITLE
Release Candidate v0.0.5

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Release Candidate for version 0.0.5.

The v0.0.5 release adds multiple new instruments, as well as improves support for xarray datasets.  There are multiple updates for compatibility with the latest versions of `cdflib`, `xarray`, and `pysat`.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

via pytest at github actions

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (include link [here](https://pysatnasa.readthedocs.io/en/rc_0_0_5/))
- [x] Upload the test distribution to the [testPyPi server](https://test.pypi.org/project/pysatNASA/0.0.5rc0/)
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages - pysatModels
- [x] Create a wiki page with the release text (put link [here](https://github.com/pysat/pysatNASA/wiki/v0.0.5))
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

**Comment by jklenzing** [comment]
Downstream package error documented at https://github.com/pysat/pysatModels/issues/128

File: README.md [hunk 1]
```diff
@@ -20,37 +20,41 @@ some examples on how to use the routines
...
 
 ```
 cd pysatNASA/
-python setup.py install
+pip install .
 ```
 
 Note: pre-1.0.0 version
 -----------------------
 pysatNASA is currently in an initial development phase and requires pysat 3.0.4.
```
Comment (aburrell) [inline README.md:56]: Is this still true?
```suggestion
pysatNASA is currently in an initial development phase and requires pysat 3.1.0.  
```

File: pysatNASA/instruments/de2_vefi.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Properties
+----------
+platform
+    'de2'
+name
+    'vefi'
+inst_id
+    None Supported
+tag
+    'dca' or 'ac'
```
Comment (aburrell) [inline pysatNASA/instruments/de2_vefi.py:33]:
```suggestion
tag
    'dca' or 'ac'
inst_id
    None Supported
```

File: pysatNASA/tests/test_instruments.py [hunk 3]
```diff
@@ -8,13 +8,24 @@
...
 
+import pysat
 import pytest
 
 # Make sure to import your instrument library here
 import pysatNASA
 
 # Import the test classes from pysat
 from pysat.tests.classes import cls_instrument_library as clslib
+from pysat.utils import testing
```
Comment (aburrell) [inline pysatNASA/tests/test_instruments.py:19]: All the pysat imports should be grouped together.  I recommend moving up pytest to the warnings block and moving all the pysat imports above the pysatNASA import.

File: pysatNASA/tests/test_instruments.py [hunk 4]
```diff
@@ -71,11 +88,85 @@ def test_load_cdflib(self, inst_dict):
...
+        test_inst, date = clslib.initialize_test_inst_and_date(inst_dict)
+        try:
+            test_inst.load(date=date, use_header=True, use_cdflib=True)
+        except ValueError as verr:
+            # Check if instrument is failing due to strict time flag
+            if str(verr).find('Loaded data') > 0:
+                test_inst.strict_time_flag = False
+                with warnings.catch_warnings(record=True) as war:
+                    test_inst.load(date=date, use_header=True, use_cdflib=True)
+                assert len(war) >= 1
```
Comment (aburrell) [inline pysatNASA/tests/test_instruments.py:121]:
```suggestion
                    test_inst.load(date=date, use_header=True, use_cdflib=True)

                assert len(war) >= 1
```

**Review by aburrell — COMMENTED** [review]
More style corrections.

**Review by aburrell — APPROVED** [review]

**Review by JonathonMSmith — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,64 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.0.5] - 2023-06-23
```
Comment (jklenzing) [inline CHANGELOG.md:5]:
```suggestion
## [0.0.5] - 2023-06-27
```

**Pull request merged**